data/indicator_3-c-1.csv

```diff
@@ -1,7 +1,5 @@
-Year,Group,Value
-2015,A,1
-2015,B,3
-2015,,2
-2016,A,1
-2016,B,3
-2016,,2
+Year,Type,Value
+2016/17,Doctors,1/10055

```

Comment:
Hi @nechri there is no feature in place too calculate values in the Value column and a fraction wouldn't be treated as a numeric (which is the only type accepted in the Value column). I noticed that the PR failed the Travis checks so I think to fix this you will need to do the calculation before hand and have that in the value column e.g. instead of having `1/10055` you would have `0.000099453`.